source/IJSVGColor.m

```diff
@@ -616,10 +616,8 @@ + (NSColor *)colorFromHEXString:(NSString *)string
         NSInteger r = (hex>>16) & 0xFF;
         NSInteger g = (hex>>8) & 0xFF;
         NSInteger b = (hex) & 0xFF;
-        return [NSColor colorWithDeviceRed:r/255.f
-                                     green:g/255.f
-                                      blue:b/255.f
-                                     alpha:alpha];
+
+        return [NSColor colorWithRed255:r green255:g blue255:b alpha255:255.0 * alpha];
```

Comment:
The colors being set were not correct, they need to map to the SRGB color space from web values.